src/content/doc-surrealdb/index.mdx

```diff
@@ -35,8 +47,105 @@ import DarkGettingStarted from "@img/image/dark/gettingstarted.png";
   }}
 />
 
-SurrealDB makes building and scaling realtime apps dramatically quicker and easier. Get started by [installing the server](/docs/surrealdb/installation), and jump into our [getting started guide](/docs/surrealdb/introduction/start) to learn how to insert and query data in SurrealDB in minutes.
+SurrealDB is a [Multi-Model Database](/features/multi-model-database) that allows you to store and manage date in relational, document, graph, time-series, vector & search and key-value models in one place. 
+
+Unified by a Powerful SQL like query language [SurrealQL](/docs/surrealql), you can query SurrealDB in ways that are familiar to you, but with the power of a modern database. In addition, SurrealDB is designed to be [distributed](/docs/surrealdb/introduction/architecture) and [scalable](/docs/surrealdb/introduction/architecture), allowing you to build and scale your applications with ease.
+
+Also providing additonal query methods like [GraphQL](/docs/surrealdb/querying/graphql), [HTTP](/docs/surrealdb/integration/http) and [RPC](/docs/surrealdb/integration/rpc), you can also query SurrealDB in your native environments using [SDKs](/docs/surrealdb/integration/sdks), we support [JavaScript](/docs/sdk/javascript), [Python](/docs/sdk/python), [Go](/docs/sdk/golang), [Rust](/docs/sdk/rust), [Java](/docs/sdk/java) and [Dotnet](/docs/sdk/dotnet).
```

Comment:
```suggestion
Also providing additional query methods like [GraphQL](/docs/surrealdb/querying/graphql), [HTTP](/docs/surrealdb/integration/http) and [RPC](/docs/surrealdb/integration/rpc), you can also query SurrealDB in your native environments using [SDKs](/docs/surrealdb/integration/sdks), we support [JavaScript](/docs/sdk/javascript), [Python](/docs/sdk/python), [Go](/docs/sdk/golang), [Rust](/docs/sdk/rust), [Java](/docs/sdk/java) and [C#](/docs/sdk/dotnet).
```

Reply:
@Odonno Just curious, aren't F# developers able to write wrappers around C# libraries? I haven't touched F# for quite a while but I seem to remember reading that technically anything in C# can be used for other dotnet languages with a bit of work.

Reply:
.NET indeed supports 3 languages: VB, C# and F#. VB is considered dead and almost never referenced. C# is the primary language while F# is loved by the community. It is technically true that it is possible to use C# libraries in an F# project but I believe that won't be fair to say it currently supports F# as it will miss important functional features.

Anyway, I changed the text because `dotnet` is not valid naming. Either use `.NET` or `C#`.